# HG changeset patch
# User Rik <rik@octave.org>
# Date 1346889559 25200
#      Wed Sep 05 16:59:19 2012 -0700
# Node ID ec3a9ff6c37529f81d56aa9674709bfda6fcc9ee
# Parent  df7ab06ff541ad03276159bbdf4c0836d89286c6
configure.ac: Use true/false rather than yes/no for build_gui variable.

* configure.ac: Use true/false rather than yes/no for build_gui variable.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2181,23 +2181,23 @@ AC_SUBST(WARN_CFLAGS)
 AC_SUBST(WARN_CXXFLAGS)
 
 ### GUI/Qt related tests.
 
 HAVE_QT=false
 QT_INCDIR=
 QT_LIBDIR=
 win32_terminal=no
-build_gui=yes
+build_gui=true
 GUIDIR=libgui
 AC_ARG_ENABLE(gui,
   [AS_HELP_STRING([--disable-gui], [don't build the GUI])],
-  [if test "$enableval" = no; then build_gui=no; fi], [build_gui=yes])
+  [if test "$enableval" = no; then build_gui=false; fi], [build_gui=true])
 
-if test $build_gui = yes; then
+if test $build_gui = true; then
   ## Check for Qt libraries
   PKG_CHECK_MODULES(QT, [QtCore, QtGui, QtNetwork],
     [],
     [AC_MSG_ERROR([Qt libraries are required to build the GUI])])
 
   ## Check for Qt utility programs
   AC_CHECK_PROGS(QMAKE, [qmake-qt5 qmake-qt4 qmake])
   if test -z "$QMAKE"; then
@@ -2261,17 +2261,17 @@ if test $build_gui = yes; then
       AC_SEARCH_LIBS([openpty], [util],
         [AC_DEFINE(HAVE_OPENPTY, [], [Define whether openpty exists])])
       ;;
   esac
 else
   ## GUI disabled.  Eliminate building GUIDIR directory
   GUIDIR=
 fi
-AM_CONDITIONAL([AMCOND_BUILD_GUI], [test $build_gui = yes])
+AM_CONDITIONAL([AMCOND_BUILD_GUI], [test $build_gui = true])
 AM_CONDITIONAL([WIN32_TERMINAL], [test $win32_terminal = yes])
 AC_SUBST(QT_INCDIR)
 AC_SUBST(QT_LIBDIR)
 AC_SUBST(GUIDIR)
 
 ### Run configure in subdirectories.
 
 export CC
